nixos/modules/system/boot/systemd/initrd.nix

```diff
@@ -508,5 +506,9 @@ in {
 
     boot.kernelParams = lib.mkIf (config.boot.resumeDevice != "") [ "resume=${config.boot.resumeDevice}" ];
 
+    system.requiredKernelConfig = with config.lib.kernelConfig; [
+      (isYes "TMPFS")
+      (isYes "BLK_DEV_INITRD")
+    ];
```

Comment:
Is this a result of not having `systemd-random-seed` in the initrd? :thinking:

Reply:
oops that was an accident :/